server/src/main/java/org/elasticsearch/action/search/SearchProgressListener.java

```diff
@@ -32,6 +32,15 @@ public abstract class SearchProgressListener {
 
     private List<SearchShard> shards;
 
+    /**
+     * Executed one when minimize round trips is enabled to indicate the number of remote clusters
```

Comment:
The beginning of the sentence has something off :)

Reply:
Fixed.